packages/dds/package.py

```diff
@@ -18,13 +18,13 @@ class Dds(CMakePackage):
     git      = "https://github.com/FairRootGroup/DDS"
 
     version('master', branch='master')
-    version('3.0', sha256='6b6bdbdeb2d43ca2917400e1503e1f5690fd9efb8b18358bd1052ee31ae6974a')
+    version('3.0', tag='3.0')
```

Comment:
Why do you want to link to a tag here?

* The line with `sha256` uses a tarball generated by github from the tagname (see the `url = ` a few lines up) and gives the hash of that tar file. This is cryptographically much more secure.
* spack can cache source tarballs much better than it can cache git checkouts. So downloading from a local mirror or even the local source cache is better that way.

Yes, in the long run, I would like to rewrite the older (2.x) versions to use that too. I just started with 3.0 to get it working with one version in this way.

Reply:
@ChristianTackeGSI , we (DDS and ODC) would prefer to use native git tags for recipes.
For example, just in case if the source repos are moved to GitLab or any other host. 
Also, what is more important, tags are human-readable and thus easier to maintain.

Reply:
Well, the readable part is the first argument to `version()`–the version number.
The sha256 is a digitial signature, not intended to be readable, really. There are spack tools to maintain that part.

If moving from github really is an issue: Then we should depend on the tar files at http://dds.gsi.de/releases/dds/ But I think, for the 3.0 release, the tar file there does not match the git tag.